packages/atomic/src/pages/examples/recommendations.html

```diff
@@ -45,17 +45,23 @@
     </script>
     <style>
       .layout {
+        padding: 20px;
         display: grid;
         grid-template-columns: 2fr 1fr;
         gap: 30px;
       }
 
-      atomic-recs-list.tiles {
+      .tiles {
         --atomic-recs-number-of-columns: 5;
       }
 
+      .carousel {
+        grid-column: 1 / span 2;
+        --atomic-recs-number-of-columns: 3;
```

Comment:
What I don't like about this implementation is the number of columns can be changed with CSS while the number of pages has to be changed with JS (if the users wants to adapt it).